src/Kunstmaan/NodeBundle/ValueObject/PageTab.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Kunstmaan\NodeBundle\ValueObject;
+
+class PageTab {
```

Comment:
value objects should not have setters, but require all of the fields in a constructor. This way you can be sure the object is always in a valid state, and there is no way for a programmer to make a mistake like this.